app/screens/Bible/VerseView.js

```diff
@@ -69,87 +55,87 @@ class VerseView extends Component {
-            : this.props.styles.verseTextSelectedNotHighlighted,
-          ]}
-          >
-          {getResultText(this.props.verseData.text)}
-        </Text> 
-        </Text>
-        {
-        (this.props.verseData.metadata && this.props.verseData.metadata[0].section) ?
-        <Text style={this.props.styles.sectionHeading}>
-          {"\n"} {this.props.verseData.metadata[0].section.text } 
-        </Text>
-        : null
-        }
-        {/* {isNoted ? <Icon name="note-outline" size={20} style={{padding:8}} /> :null}  */}
-          </Text>
-        )
-      }
-        return (
-          <Text style ={this.props.styles.textStyle} onPress={() => {this.onPress()}} >
-            <Text>
-              <Text style={this.props.styles.verseNumber}>
-                {this.props.verseData.number}{" "}
+    if (this.props.verseData.number == 1) {
+      return (
+        <Text style={this.props.styles.textStyle}>
+          {
+            this.props.chapterHeader != null ?
+              <Text style={this.props.styles.sectionHeading}>
+                {this.props.chapterHeader} {"\n"}
               </Text>
-              <Text style={[isSelect && isHighlight 
-                ? this.props.styles.verseTextSelectedHighlighted 
-                : !isSelect && !isHighlight 
+              :
+              null
+          }
+          <Text onPress={() => { this.onPress() }}>
+            <Text style={this.props.styles.verseChapterNumber}>
+              {this.props.chapterNumber}{" "}
+            </Text>
+            <Text style={[isSelect && isHighlight
+              ? this.props.styles.verseTextSelectedHighlighted
+              : !isSelect && !isHighlight
                 ? this.props.styles.verseTextNotSelectedNotHighlighted
                 : !isSelect && isHighlight
-                ? this.props.styles.verseTextNotSelectedHighlighted
-                : this.props.styles.verseTextSelectedNotHighlighted,
-              ]}
-              >
-                {getResultText(this.props.verseData.text) } 
-              </Text>  
+                  ? this.props.styles.verseTextNotSelectedHighlighted
+                  : this.props.styles.verseTextSelectedNotHighlighted,
+            ]}
+            >
+              {getResultText(this.props.verseData.text)}
             </Text>
-            {
+          </Text>
+          {
             (this.props.verseData.metadata && this.props.verseData.metadata[0].section) ?
               <Text style={this.props.styles.sectionHeading}>
-                {"\n"} {this.props.verseData.metadata[0].section.text } 
+                {"\n"} {this.props.verseData.metadata[0].section.text}
               </Text>
-            : null
-            }
-            {/* {isNoted ? <Icon name="note-outline" size={20} style={{padding:8}} /> :null}  */}
+              : null
+          }
+          {/* {isNoted ? <Icon name="note-outline" size={20} style={{padding:8}} /> :null}  */}
```

Comment:
remove the commented code if not required

Reply:
It is required for notes touchable for verses